Documentation/doc/resources/1.8.13/BaseDoxyfile.in

```diff
@@ -78,7 +78,7 @@ ALIASES                = "cgal=%CGAL" \
                          "gnu=GNU" \
                          "ms=MS" \
                          "qt=Qt" \
-                         "qt5=Qt5" \
+                         "qt6=Qt6" \
```

Comment:
This change should also be done in the, on November 15 2023, integrated, 1.10.0 version (see warnings in the overnight documentation build https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.0-Ic-108/logs_master/Documentation.log

Reply:
I note this as a TODO (see the top message).